bus-mapping/src/circuit_input_builder.rs

```diff
@@ -1524,9 +1524,17 @@ impl<'a> CircuitInputBuilder {
         eth_block: &EthBlock,
         geth_traces: &[eth_types::GethExecTrace],
     ) -> Result<(), Error> {
+        // record tx gas cumulative used
+        let mut tx_cumulative_gas: HashMap<usize, u64> = HashMap::new();
```

Comment:
On a different subject: It looks like `tx_cumulative_gas` accumulates gas across all txs in the block, should we rename `tx_cumulative_gas` to `block_cumulative_gas`?

Reply:
yes, it accumulates gas across all txs in the block !  since by default we handle txs in one block,  Can we omit the `block` prefix or   `block_tx_cumulative_gas` which sounds good to me , what do you think @ChihChengLiang ?

Reply:
My opinion would be to simply use `cumulative_gas_used`. The `tx_` is definitely misleading.

Reply:
use  `cumulative_gas_used`  in https://github.com/appliedzkp/zkevm-circuits/pull/466/commits/db106a33e264ff347da007fa2028dfc9ee8d0063 thanks